projects/demo-maps/src/test.ts

```diff
@@ -1,13 +1,19 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
 
+import 'zone.js/dist/zone';
```

Comment:
There are several cases where we're importing `zone` alongside `testing-zone` for our unit-tests, but I don't see changes to the actual tests. Is this a CI requirement? Is it because we're no longer using `karma-coverage-istanbul`? Or because of those new async methods in the e2e tests?

Reply:
Also generated by angular.

Reply:
Thanks!